processing/src/main/java/org/apache/druid/segment/virtual/ExpressionVirtualColumn.java

```diff
@@ -393,7 +393,7 @@ private boolean isDirectAccess(final ColumnInspector inspector)
   {
     if (parsedExpression.get().isIdentifier()) {
       final ColumnCapabilities baseCapabilities =
-          inspector.getColumnCapabilities(parsedExpression.get().getBindingIfIdentifier());
+          inspector.getColumnCapabilities(parsedExpression.get().getIdentifierIfIdentifier());
```

Comment:
was this related to this patch somehow?

Reply:
no, this was an unintentional change, reverted